plugins/woocommerce-admin/client/products/sections/product-variation-details-section.tsx

```diff
@@ -59,6 +65,12 @@ export const ProductVariationDetailsSection: React.FC = () => {
 
 	const imageFieldProps = getInputProps( 'image' );
 
+	useEffect( () => {
+		if ( parent?.images?.length && ! values.image ) {
```

Comment:
Sorry I did not update the AC on the original issue, but I think we can skip this portion and keep an empty image field if none is added for a variation.

This is the behavior in the legacy experience and I'm a little hesitant to automatically make the variation image the same as the parent.  But cc @jarekmorawski in case you don't like this route.

Reply:
I think it's okay. We can come back to this at a later stage. Ideally, we would consistently copy all parent's data to the variations. For example, if we show the price and description, we should show the image, too.